rt_mcca.m

```diff
@@ -109,12 +109,12 @@
 unmixing_flipped = unmixing;
 for k = 1:nstory
   fprintf('aligning polarity across channels for story %d/%d\n', k, nstory);
-  input = reshape(unmixing(:,:,:,k), size(unmixing,1), []);
```

Comment:
Oh, I guess the switch between (un)mixing matrix was due to testing reasons.

Reply:
that was based on some trying out, at the moment both unmixing and mixing give unsatisfactory results.